website/src/documentation/runtime/environments.md

```diff
@@ -0,0 +1,187 @@
+$USER-docker-apache.bintray.io/beam/java       latest             0103512f1d8f     34 minutes ago      780MB
+$USER-docker-apache.bintray.io/beam/go         latest             ce055985808a     35 minutes ago      121MB
+```
+
+<b>Although the respository names look like URLs, the container images are stored locally on your workstation.</b> After building the container images locally, you can [push](#pushing-container-images) them to an eponymous repository online.
+
+### Overriding default Docker targets
+
+The default SDK version is `latest` and the default Docker repository is the following Bintray location:
+
+```
+$USER-docker-apache.bintray.io/beam
+```
+
+When you [build SDK container images](#building-container-images), you can override the default version and location.
+
+To specify an older Python SDK version, like 2.3.0, build the container with the `docker-tag` option:
+
+```
+./gradlew docker -Pdocker-tag=2.3.0
+```
+
+To change the `docker` target, build the container with the `docker-repository-root` option:
+
+```
+./gradlew docker -Pdocker-repository-root=$LOCATION
+```
+
+## Customizing container images
+
+You can add extra dependencies or serialization files to container images so the execution engine doesn't need them.
+
+To customize a container image, either:
+* [Write a new](#writing-new-dockerfiles) [Dockerfile](https://docs.docker.com/engine/reference/builder/) on top of the original
+* [Modify](#modifying-dockerfiles) the [original Dockerfile](https://github.com/apache/beam/blob/master/sdks/python/container/Dockerfile) and reimage the container
+
+It's often easier to write a new Dockerfile, but you can customize anything, including the base OS, by modifying the original.
+
+### Writing new Dockerfiles on top of the original {#writing-new-dockerfiles}
+
+<ol>
+    <li>
+        Pull a <a href="https://console.cloud.google.com/gcr/images/apache-beam-testing/GLOBAL/beam/sdks/release">prebuilt SDK container image</a> for your target language and version.
+    </li>
+    <li>
+        <a href="https://docs.docker.com/develop/develop-images/dockerfile_best-practices/">Write a new Dockerfile</a> that <a href="https://docs.docker.com/engine/reference/builder/#from">designates</a> the original as its <a href="https://docs.docker.com/glossary/?term=parent%20image">parent</a>
+    </li>
+    <li>
+        Build a child image: <pre>docker build -f /path/to/new/Dockerfile</pre>
+    </li>
+</ol>
+
+### Modifying the original Dockerfile {#modifying-dockerfiles}
+
+1. Pull the [prebuilt SDK container image](https://console.cloud.google.com/gcr/images/apache-beam-testing/GLOBAL/beam/sdks/release) for your target language and version
+2. Customize the [Dockerfile](https://github.com/apache/beam/blob/master/sdks/python/container/Dockerfile). If you're adding dependencies from [PyPI](https://pypi.org/), use `base_image_requirements.txt` instead of the Dockerfile.
```

Comment:
Let's add a link to `base_image_requirements.txt`, which is
`https://github.com/apache/beam/blob/master/sdks/python/container/base_image_requirements.txt`